### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/lgpl-docs/pull/48

cc @GuillaumeGomez, @sdroege

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
:+1:

**Comment by GuillaumeGomez** [comment]
@sdroege You can merge now. :p

**Comment by sdroege** [comment]
It's still marked as WIP

**Comment by EPashkin** [comment]
It still need right gir-files

File: src/auto/functions.rs [hunk 1]
```diff
@@ -916,6 +930,41 @@ pub fn random_set_seed(seed: u32) {
     }
 }
 
+//#[cfg(any(feature = "v2_58", feature = "dox"))]
+//pub fn rc_box_acquire(mem_block: /*Unimplemented*/Fundamental: Pointer) -> /*Unimplemented*/Option<Fundamental: Pointer> {
```
[the remark this answers is not in the document]
Comment (EPashkin) [reply src/auto/functions.rs]: I will do it

**Comment by EPashkin** [comment]
PR updated, must pass CI

**Comment by EPashkin** [comment]
CI passed

**Pull request merged**